qutebrowser/browser/commands.py

```diff
@@ -1076,10 +1076,11 @@ def tab_focus(self, index: typing.Union[str, int]=None, count=None):
                    last tab.
             count: The tab index to focus, starting with 1.
         """
-        if index == 'last':
+        index = count if count is not None else index
+
+        if index in ('last', self._current_index() + 1):
             self._tab_focus_last()
```

Comment:
As for not raising the error in the `self._current_index() + 1` case - what about this?

```python
if index == 'last':
    self._tab_focus_last()
    return
elif index == self._current_index() + 1:
    self._tab_focus_last(show_error=False)
    return
elif index is None:  # from below
    self.tab_next()
    return
```

Then in `_tab_focus_last`, add a `show_error` argument, and just return instead of raising `CommandError` if it's false.